crates/revm/src/state_change.rs

```diff
@@ -36,40 +36,34 @@ use std::collections::HashMap;
 ///
 /// Balance changes might include the block reward, uncle rewards, withdrawals, or irregular
 /// state changes (DAO fork).
-#[allow(clippy::too_many_arguments)]
 #[inline]
 pub fn post_block_balance_increments(
     chain_spec: &ChainSpec,
-    block_number: u64,
-    block_difficulty: U256,
-    beneficiary: Address,
-    block_timestamp: u64,
+    block: &BlockWithSenders,
```

Comment:
I don't think we need the senders here, right?

can we just accept `&Block` via deref:

https://github.com/paradigmxyz/reth/blob/9fd2cf027f130e14d75ca6eecb24b13ab6dec1ba/crates/primitives/src/block.rs#L198-L202

Reply:
fixed